background/logging.js

```diff
@@ -0,0 +1,32 @@
+require('app-module-path').addPath(__dirname + '../../app');
+
+var path = require('path');
+
+var nconf = require('nconf');
+var bunyan = require('bunyan');
+
+formatBunyanInfoLog = function() {};
+formatBunyanInfoLog.prototype.write = function (rec) {
+    console.log('[%s] %s: %s',
+        rec.time,
+        bunyan.nameFromLevel[rec.level],
+        rec.msg);
+};
+
+logger = bunyan.createLogger({
+    name: 'Bitcannon',
```

Comment:
Would it be possible for the log.warn, etc. to add a custom field? We could then have `threadType` and have that set as webui, importer, etc.